contributors/design-proposals/kep-template.md

```diff
@@ -0,0 +1,244 @@
+   - [How Do I Teach This?](#how-do-i-teach-this)
+      - [KEP Template](#kep-template-1)
+      - [Section: Metadata](#section-metadata)
+      - [Section: Table of contents](#section-table-of-contents)
+      - [Section: Summary](#section-summary)
+      - [Section: Motivation](#section-motivation)
+      - [Section: Examples](#section-examples)
+      - [Section: How Do I Teach This?](#section-how-do-i-teach-this)
+      - [Section: Detailed Design](#section-detailed-design)
+      - [Section: Graduation Criteria](#section-graduation-criteria)
+      - [Section: Implementation History](#section-implementation-history)
+      - [Section: Drawbacks](#section-drawbacks)
+      - [Section: Alternatives](#section-alternatives)
+      - [Section: Unresolved Questions](#section-unresolved-questions)
+      - [Section: Mentors](#section-mentors)
+   - [Graduation Criteria](#graduation-criteria)
+   - [Drawbacks](#drawbacks)
+   - [Alternatives](#alternatives)
+   - [Unresolved Questions](#unresolved-questions)
+   - [Mentors](#mentors)
+
+## Summary
+
+The KEP template combines our design proposals with functionality currently
+provided by the [features repo][] within a flat file in source control. SIGs
+will adopt and fork the KEP template to support their specific needs.
+
+[features repo]: https://github.com/kubernetes/features
+
+## Motivation
+
+A template for a unit of work described in a [proposal to standardize][] the
+Kubernetes development process is provided.
+
+[proposal to standardize]: https://github.com/kubernetes/community/pull/967
+
+## Examples
+
+- [this KEP][]
+- [the KEP process][]
+
+[this KEP]: https://github.com/calebamiles/community/blob/propose-kep-template/contributors/design-proposals/kep-template.md
+[the KEP process]: https://github.com/kubernetes/community/pull/967
+
+## Detailed Design
+
+## How Do I Teach This?
+
+[The KEP process][] proposal will outline how a KEP is expected to progress
+from conception to implementation in one or more pull requests to a Kubernetes
+repository. This suggests that a KEP template should be used to track work
+which ultimately will result in change to source control; if you find success
+in modifying the KEP template please notify the [mentors](#mentors)!
+
+### KEP Template
+
+In order to implement an KEP like process the following template will be created
+
+```
+# Title
+## Metadata
+## Table of Contents
+## Summary
+## Motivation
+## Examples [optional]
+## How Do I Teach This? [optional]
+## Detailed Design
+## Graduation Criteria
+## Implementation History
+## Drawbacks [optional]
+## Alternatives [optional]
+## Unresolved Questions [optional]
+## Mentors [optional]
+```
+
+where sections marked `[optional]` may be omitted by KEP authors. It is expected
+that most KEPs will begin their lives in discussion with the responsible SIGs,
+preferably in a mailing list so that the discussion is saved for future Kubernauts.
+
+
+### Section: Metadata
```

Comment:
This nesting is confusing. I was expecting a template that could be copied and filled in.

Reply:
extracted empty template from section explanations